docker-compose/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.0'
 
 services:
   db:
-    image: timescaledev/timescaledb-ha:pg12-latest
+    image: timescale/timescaledb-ha:pg14-latest
```

Comment:
Shouldn't this also use the promscale-extension image? Especially now that we're planning to require the extension, might be good to already have these files depend on that image.

Reply:
timescaledb-ha also has the extension packaged.

Reply:
Yep its already there